Tribler/community/multichain/community.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, *args, **kwargs):
         self._private_key = self.my_member.private_key
         self._public_key = self.my_member.public_key
         self.persistence = MultiChainDB(self.dispersy, self.dispersy.working_directory)
+        self.logger.debug("The multichain community started with Public Key: " + base64.encodestring(self._public_key))
```

Comment:
I looked into lazy logging and I see what you mean. I think that style looks really ugly, but I guess it makes sense performance-wise.